doc/dev/gravel/systemd-boot.md

```diff
@@ -0,0 +1,45 @@
+# Aquarium systemd boot process
+
+## aquarium-boot
+
+`systemd` unit that will be running after `local-fs.target`, runs
+`/usr/share/aquarium/boot/aqrbootsetup.sh`, which tries to find an lvm disk
```

Comment:
```suggestion
`/usr/share/aquarium/boot/aqrbootsetup.sh`, which tries to find a LVM disk
```

Reply:
I'm fine with the "lvm" -> "LVM" change, but, and may @asettle correct me, it's still an "an LVM" on the account that it's read "an ell-v-m".

Reply:
LVM :heavy_check_mark: 
an :heavy_check_mark: